plat/kvm/arm/setup.c

```diff
@@ -163,7 +167,11 @@ void __no_pauth _ukplat_entry(struct ukplat_bootinfo *bi)
 
 	fdt = (void *)bi->dtb;
 
+#if CONFIG_LIBUKRSI
```

Comment:
Please remember to add the comment here, as stated in the previous feedback round.